Sources/Core/UIKit/Other/UndoManagerMonitor.swift

```diff
@@ -0,0 +1,241 @@
+//
+//  UndoManagerMonitor.swift
+//  XestiMonitors
+//
+//  Created by Rose Maina on 2018-04-30.
+//
+//  © 2018 J. G. Pusey (see LICENSE.md)
+//
+
+#if os(iOS) || os(macOS) || os(tvOS) || os(watchOS)
+
+import Foundation
+import UIKit
```

Comment:
`UIKit` is _not_ used by this monitor and will, in fact, not compile on macOS and watchOS.